Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMElementGtk.cpp

```diff
@@ -821,18 +821,10 @@ void webkit_dom_element_scroll_into_view_if_needed(WebKitDOMElement* self, gbool
 
 void webkit_dom_element_scroll_by_lines(WebKitDOMElement* self, glong lines)
```

Comment:
Should these functions be removed instead of left as NOPs, or will that break the ABI?

Reply:
hehe 
![image](https://github.com/WebKit/WebKit/assets/505230/98f36b15-ca20-47d9-9b7e-f30cc74b68ca)

see the discussion in https://github.com/WebKit/WebKit/pull/20743#discussion_r1399398481